ruma-client-api/src/error/kind_serde.rs

```diff
@@ -209,71 +209,40 @@ impl<'de> Visitor<'de> for ErrorKindVisitor {
 }
 
 // FIXME: Add `M_FOO_BAR` as a naming scheme in StringEnum and remove rename attributes.
-#[derive(FromString, DeserializeFromCowStr)]
+#[derive(Clone, Debug, PartialEq, Eq, StringEnum)]
```

Comment:
Please leave the derive list as-is, all the extra trait implementations aren't needed for this (private) type.

Reply:
Please just restore `FromString, DeserializeFromCowStr`.